scripts/blocks-utils.js

```diff
@@ -95,3 +95,50 @@ export function addTabs(tabs, block) {
     }
   });
 }
+
+export function setMetaTag(tagType, propertyKey, propertyValue, url) {
+  const tag = document.querySelector(`${tagType}[${propertyKey}='${propertyValue}']`);
+  if (tag) {
+    if (tagType === 'link') {
+      tag.href = url;
+    } else {
+      tag.content = url;
+    }
+  } else {
+    const meta = document.createElement(tagType);
+    meta.setAttribute(propertyKey, propertyValue);
+    if (tagType === 'link') {
+      meta.href = url;
+    } else {
+      meta.content = url;
+    }
+    document.head.appendChild(meta);
+  }
+}
+
+export function setOgImages(picture) {
+  if (picture && picture.outerHTML) {
+    const imageUrl = window.location.origin + picture.outerHTML.match(/<img [^>]*src="[^"]*"[^>]*>/gm)
+      .map((x) => x.replace(/.*src="([^"]*)".*/, '$1'));
+    if (imageUrl) {
+      const OgTags = ['og:image', 'og:image:secure_url'];
+      OgTags.forEach((tag) => {
+        setMetaTag('meta', 'property', tag, imageUrl);
+      });
+      setMetaTag('meta', 'name', 'twitter:image', imageUrl);
+    }
+  }
+}
+
+export function setTagPageTitles(tagTitle) {
+  if (tagTitle) {
+    window.document.title = tagTitle;
```

Comment:
Maybe we should capitalise this title, it feels a bit off for certain tags like `germany`, the breadcrumb would benefit as well

https://tagmeta--sunstar--hlxsites.hlx.live/healthy-thinking/tag?feed-tags=germany 

<img width="847" alt="Screenshot 2023-11-02 at 11 41 06" src="https://github.com/hlxsites/sunstar/assets/44490413/0e8bd7e8-7b7d-40b4-96ec-ff94e493c93e">

Reply:
@dnbute thanks for the feedback. I just keep the same with the current site and it shows the tag title not the tag id. It's stored in [placeholders.xls](https://adobe.ly/49bpodU), we can update it anytime.

You can check the current page here - https://www.sunstar.com/healthy-thinking/tag/germany/
<img width="747" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/f84b4d84-e320-4df2-bc45-d5196af3f8c8">

Reply:
Even better if we can control it from the sheet 👍

Reply:
`setTagPageTitles` looks specific to `tags` block and csan be moved there as this block-utils seems to be the place to have code that's reusable across blocks.